### PR TITLE
Python binding

### PR DESCRIPTION
**Pull request by qnzhou**
Enable github workflow to build and deploy python binding.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    "wheel>=0.36.0",
+    "scikit-build==0.14.0",
+    "cmake>=3.24.2",
+    "ninja; platform_system!='Windows'"
+]
+
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "lagrange-open"
```
Comment (jdumas) [inline pyproject.toml:13]: Not `lagrange-adobe`?

File: .github/workflows/wheel.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
+name: Python wheel
+
+on:
+  push:
+    tags:
+      - '*'
```
Comment (jdumas) [inline .github/workflows/wheel.yaml:6]: This seems to be triggered by regular commits on this PR, is that expected? Is this action going to publish draft packages first? Can we use a staging pypi repo to test this github actions, or run it with a dry-run option at first?

File: pyproject.toml (hunk 1 above)
Comment (qnzhou) [reply pyproject.toml:13]: I don't have strong opinions, but `lagrange-open` seems to be more consistent with our naming convention.

File: .github/workflows/wheel.yaml (hunk 2 above)
Comment (qnzhou) [reply .github/workflows/wheel.yaml:6]: This workflow is triggered when a new tag is pushed, not by every commit.  I am manually pushing tags for each commit for testing purposes.  We can also delete any botched release on PyPI, so I am not too worried about that.

**Review by jdumas — APPROVED** [review]

**Pull request merged**